### PR TITLE
[typo](doc) fix error description about load_to_single_tablet

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

Fix error description about load_to_single_tablet.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run fe-ut

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: docs/en/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/BROKER-LOAD.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ WITH BROKER broker_name
     
   - `load_to_single_tablet`
   
-    Boolean type, True means that one task can only load data to one tablet in the corresponding partition at a time. The default value is false. The number of tasks for the job depends on the overall concurrency. This parameter can only be set when loading data into the OLAP table with random partition.
+    Boolean type, True means that one task can only load data to one tablet in the corresponding partition at a time. The default value is false. The number of tasks for the job depends on the overall concurrency. This parameter can only be set when loading data into the OLAP table with random bucket.
```
Comment (freemandealer) [inline docs/en/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/BROKER-LOAD.md:197]: maybe use the gerund form 'bucketing'  to emphasize it is a verb?

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**